packages/components/quick-filters/src/quick-filters.tsx

```diff
@@ -1,12 +1,57 @@
+import Tag from '@commercetools-uikit/tag';
+import { css } from '@emotion/react';
+import { designTokens } from '@commercetools-uikit/design-system';
+
+export type TQuickFiltersItem = {
+  /** unique identifier for the item. */
+  id: string;
+  /* label to display */
+  label: string;
+  /* the current active state of the item */
+  isActive: boolean;
+};
+
 export type TQuickFiltersProps = {
   /**
-   * This is a stub prop!
+   * collection of quick filter items
+   *
+   * @param item.id unique identifier for the item.
+   * @param item.label label to display
+   * @param item.isActive the current active state of the item
```

Comment:
Not ideal, since the descriptions are duplicated, but this way the expected item-shape shows up directly in the components' props-table.

Reply:
not ideal, but definitely helpful and something we should do in `Filters` as well